### PR TITLE
Split analytic* into analytic_account* to isolate dependency on account (closes #3)

### LINKED ISSUE #3: Remove `analytic_department` dependency on `account`
Right now it forces you to install Accounting, but you may want to use it only with Projects, without having Accounting installed.

### PR DESCRIPTION
**Pull request by dreispt**
- The new module `analytic_base_department` adds the `department_id` to Analytic Account, and only depends on `analytic`
- `analytic_department` depends on `analytic_base_department` AND `account`, thus keeping the same behaviour as before.

### CODE REVIEW

**Comment by dreispt** [comment]
Making a clearer explanation on this:

The original `analytic_department` included an excessive dependency on `account`
This splits part of the module into `analytic_base_department`: 
- if you install `analytic_department`, you will get the same as before
- if you install `analytic_base_department` you will get only those changes on Analytic Accounts

File: analytic_base_department/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+import analytic
```
Comment (pedrobaeza) [inline analytic_base_department/__init__.py]: `from . import analytic`

File: analytic_base_department/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "Analytic Department Categorization",
+    "version": "2.0",
```
Comment (pedrobaeza) [inline analytic_base_department/__openerp__.py]: Why 2.0? It's the first version of this module

File: analytic_base_department/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+"X-Launchpad-Export-Date: 2011-04-16 07:04+0000\n"
+"X-Generator: Launchpad (build 12735)\n"
+
+#. module: analytic_department
+#: field:account.analytic.account,department_id:0
+#: view:account.analytic.account:0
+#: field:account.analytic.line,department_id:0
+#: view:account.analytic.line:0
+msgid "Department"
+msgstr ""
```
Comment (pedrobaeza) [inline analytic_base_department/i18n/es.po]: Please put 'Departamento'.

**Comment by dreispt** [comment]
Fixed.

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by dreispt** [comment]
Just need one more OK...
@jgrandguillaume I believe C2C uses this, right?

**Comment by dreispt** [comment]
Just one more review needed here. Anyone please?

**Comment by pedrobaeza** [comment]
As this PR has my review and Travis says green, I'll proceed with the merge.

**Pull request merged**